src/types/parsers/tree-parser.ts

```
@@ -0,0 +1,152 @@
+import type {
+  BufferGeometry,
+  Group,
+  AnimationClip,
+  MeshStandardMaterial,
+  Texture,
+  Object3D,
+  LoadingManager,
+  TextureLoader,
+} from 'three';
+import type {
+  IParsingContext,
```

ESLint

[error] 'IParsingContext' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)